dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -430,33 +432,25 @@ class InterfaceMgr(Manager):
             "CONFIG_DB",
             interface_table
         )
-        self.interfaces = {}
 
     def set_handler(self, key, data):
-        # There are two entries for an interface in CFG_INTF_TABLE.
-        # One of them is to specify ip and prefix.
-        # In this case, the key contains '|', like "Ethernet0|192.168.0.6/30".
-        # Another one is to specify whether this interface belongs to a vnet
-        # In this case, the key only includes the interface name
```

Comment:
you have removed the comments, can you add new comments? what is not correct in the comments.